_meta/fields.yml

```diff
@@ -2,8 +2,34 @@
   title: mysqlbeat
   description:
   fields:
-    - name: counter
-      type: long
+    - name: type
+      type: keyword
+      required: true
+      description: >
+        Type of reccord
+
+    - name: server
```

Comment:
`hostname` is more descriptive. (In the config a server is a collection of fields, this is the hostname field)